src/google/protobuf/compiler/java/java_message.cc

```diff
@@ -124,7 +124,7 @@ void ImmutableMessageGenerator::GenerateStaticVariables(io::Printer* printer) {
 
     // The descriptor for this type.
     printer->Print(vars,
-      "$private$static final com.google.protobuf.Descriptors.Descriptor\n"
+      "$private$static com.google.protobuf.Descriptors.Descriptor\n"
```

Comment:
Also could you update the author of the first commit to you (i.e., fizbin)? Our CLA bot found the CLA for Daniel Martin but complained that we need Daniel Martin's consent for you to commit on his behalf. We need Daniel Martin to reply on this thread, or you can just update the commit author.